src/asmcnc/comms/router_machine.py

```diff
@@ -181,6 +186,24 @@ def __init__(self, win_serial_port, screen_manager, settings_manager, localizati
         self.TMC_motor[TMC_Y2] = motors.motor_class(TMC_Y2)
         self.TMC_motor[TMC_Z] = motors.motor_class(TMC_Z)
 
+    Z_AXIS_ACCESSIBLE_ABS_HEIGHT = -5
+    Z_PROBE_SAFE_PULL_OFF = 1
```

Comment:
I would give the cutter a bit more clearance. You have to pull the mirror out and 1mm is quite tight. I'd go with 5